openmdao/drivers/genetic_algorithm_driver.py

```diff
@@ -219,6 +219,16 @@ def run(self):
             lower_bound[i:j] = meta['lower']
             upper_bound[i:j] = meta['upper']
 
+        # Initial Design Vars
+        x0 = np.empty(count)
+        desvar_vals = self.get_design_var_values()
+        i = 0
+        for name, meta in iteritems(self._designvars):
+            size = meta['size']
+            x0[i:i + size] = desvar_vals[name]
+            i += size
+
+
```

Comment:
You can move this into the previous loop with the cached indices so that you don't have to sum the sizes again. So it'll look like x0[i:j] = desvar_vals[name].

Looks good otherwise. You might need to remove the extra carriage return here to pass the lint tests.